scripts/update-nsis-functions.sh

```diff
@@ -39,8 +39,7 @@ variables=$(\
 variables_extra='{nsisdir} 0 1 2 3 4 5 6 7 8 9 r0 r1 r2 r3 r4 r5 r6 r7 r8 r9 \\n \\r \\t $'
 variables="${variables_extra} ${variables}"
 # prefix each element with a dollar sign
-declare -a variables_array=(${variables})
-variables="${variables_array[@]/#/'$'}"
+variables="$(echo $variables | sed 's/[^ ]*/$&/g')"
```

Comment:
you should quote `$variable` to avoid any unexpected things happening.

Reply:
Good catch, with quoting it turned out that there was trailing whitespace.

Reply:
and there are none in the functions list?

Reply:
There were, of course.
Removed in https://github.com/geany/geany/pull/2181/commits/1fa5f66b2f2dd772a2cb5c6b5010bc19ce3bb824.

I will squash the latest commits and merge then.